README.md

```diff
@@ -13,8 +13,9 @@ oc adm must-gather --image=registry.redhat.io/odf4/odf-must-gather-rhel9:v4.13 -
 
 List of arguments that can be passed to the odf-must-gather are:
 ```
+-o,  --odf                Collect ODF logs
 -d,  --dr                 Collect DR logs
--pc,  --provider          Collect openshift-storage-client logs from a provider/consumer cluster
+-pc, --provider           Collect openshift-storage-client logs from a provider/consumer cluster
```

Comment:
As discussed with Neha, we can remove this for now. We won't be documenting for provider/consumer this release. So once it is GAd and released, we will add this in documentations.

Reply:
Alright :)